pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -794,6 +794,7 @@ func (p *MasqueradePodInterface) createNatRulesUsingIptables(protocol iptables.P
 			strings.ToLower(port.Protocol),
 			"--dport",
 			strconv.Itoa(int(port.Port)),
+			"--source", getLoopbackAdrress(protocol),
```

Comment:
I think it is enough to specify here the keyword `localhost`.

Reply:
Thanks, I will try in on a different PR.